src/ducks/account/LoanPercentageLine.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PercentageLine from 'cozy-ui/transpiled/react/PercentageLine'
+import cx from 'classnames'
+import styles from 'ducks/account/LoanPercentageLine.styl'
+
+const LoanPercentageLine = props => {
+  const { className, ...rest } = props
+
+  return (
+    <PercentageLine
+      className={cx(styles.LoanPercentageLine, className)}
+      color="var(--emerald)"
+      {...rest}
+    />
+  )
+}
```

Comment:
Should be replaced by a PercentageBar from cozy-ui right ? Can you add a comment in this case ?